tests/test_update_manager_specific_lines.py

```
@@ -0,0 +1,242 @@
+                    mock_usda.fetch_all_foods = AsyncMock(return_value=[])
+
+                    # Execute update - should trigger exception logging
+                    result = await manager._update_usda_database(force=True)
+
+                    # Should succeed despite backup load exception
+                    assert isinstance(result, UpdateResult)
+                    assert result.success is True
+                    assert result.source == "usda"
+
+    @pytest.mark.asyncio
+    async def test_backup_load_exception_logging_lines_474_475(self, manager, temp_dir):
```

Ruff (0.14.1)

Unused method argument: `temp_dir`

(ARG002)